src/parseInlineTags.js

```diff
@@ -0,0 +1,72 @@
+
+/**
+ * @param {RegExpMatchArray} match An inline tag regexp match.
+ * @returns {string}
+ */
+function determineTextStyle (match) {
+  const {separator, text} = match.groups;
+  const [, textEnd] = match.indices.groups.text;
+  const [tagStart] = match.indices.groups.tag;
+  if (!text) {
+    return 'plain';
+  } else if (separator === '|') {
+    return 'pipe';
+  } else if (textEnd < tagStart) {
+    return 'prefix';
```

Comment:
@brettz9 your original proposal was `markdown` for this, but since markdown surrounds the right-hand-side with parentheses and not curly brackets like we're doing here, I though prefix (to signal that the text is to the left of the tag) would be more appropriate. I'd be happy to align with whatever you think is a better name for this.

Reply:
Sure, that's fine.